clang/lib/DPCT/ASTTraversal.cpp

```diff
@@ -1089,6 +1089,12 @@ void IncludesCallbacks::InclusionDirective(
     }
   }
 
+  // Always keep include of CL/opencl.h
+  // Only CUDA code is migrated, not OpenCL.
+  // Thus CL/opencl.h header must be kept
+  if (FileName == "CL/opencl.h")
```

Comment:
can the patch more generic, works for all CL/* files,  not specidifc opencl.h file. 
There are other file in CL/ should be keep, like.
cl_egl.h  cl_ext.h  cl_gl_ext.h  cl_gl.h  cl.h  cl.hpp  cl_platform.h  opencl.h

Reply:
suggest to compare "CL/ "folder, if so, keep it.

Reply:
Updated patch to keep includes of all CL* files.  Test updated also.